result = divide(8, 0); // This will throw an exception
        std::cout << "Result: " << result << std::endl; // This won't be executed